tests/foreman/ui/test_user.py

```diff
@@ -1168,3 +1056,57 @@ def test_positive_mails_for_new_timezone(self):
 
         @CaseLevel: Integration
         """
+
+
+class ActiveDirectoryUserTestCase(UITestCase):
+    """Implements Active Directory feature tests for user in UI."""
+
+    @classmethod
+    @skip_if_not_set('ldap')
+    def setUpClass(cls):  # noqa
+        super(ActiveDirectoryUserTestCase, cls).setUpClass()
+        cls.ldap_user_name = settings.ldap.username
+        cls.ldap_user_passwd = settings.ldap.password
+        cls.base_dn = settings.ldap.basedn
+        cls.group_base_dn = settings.ldap.grpbasedn
+        cls.ldap_hostname = settings.ldap.hostname
+        cls.usergroup_name = gen_string('alpha')
+
+        authsource_attrs = entities.AuthSourceLDAP(
+            onthefly_register=True,
+            account=cls.ldap_user_name,
+            account_password=cls.ldap_user_passwd,
+            base_dn=cls.base_dn,
+            groups_base=cls.group_base_dn,
+            attr_firstname=LDAP_ATTR['firstname'],
+            attr_lastname=LDAP_ATTR['surname'],
+            attr_login=LDAP_ATTR['login_ad'],
+            server_type=LDAP_SERVER_TYPE['API']['ad'],
+            attr_mail=LDAP_ATTR['mail'],
+            name=gen_string('alpha'),
+            host=cls.ldap_hostname,
+            tls=False,
+            port='389',
+        ).create()
+        cls.ldap_server_name = authsource_attrs.name
+
+    @tier2
+    def test_positive_create_in_ldap_mode(self):
+        """Create User in ldap mode
+
+        @id: 0668b2ca-831e-4568-94fb-80e45dd7d001
+
+        @Assert: User is created without specifying the password
+
+        @CaseLevel: Integration
+        """
+        user_name = gen_string('alpha')
+        with Session(self.browser) as session:
+            make_user(
+                session,
+                username=user_name,
+                authorized_by='LDAP-' + self.ldap_server_name,
+                password1='',
+                password2='',
```

Comment:
Not sure what you meant. They are optional and in case they are not passed, value generated automatically to prevent huge code duplication for all test cases. Here is a goal to pass empty values to verify specific functionality